src/backend/utils/wiki-feed-parser.js

```diff
@@ -34,33 +31,29 @@ module.exports.parseData = function() {
 
   let line = '';
 
-  return this.getData()
-    .then(data => {
-      const objArray = [];
-      let feed = [];
-      data.forEach(element => {
-        if (!commentCheck.test(element)) {
-          if (element.startsWith('[')) {
-            // eslint-disable-next-line no-useless-escape
-            line = element.replace(/[\[\]']/g, '');
-            feed.push(`${line}`);
-          }
-          if (nameCheck.test(element)) {
-            line = element.replace(/^\s*name\s*=\s*/, '');
-            feed.push(`${line}`);
-            let obj = {
-              name: feed[feed.length - 1],
-              url: feed[feed.length - 2],
-            };
-            objArray.push(obj);
-            feed = [];
-            obj = {};
-          }
+  return this.getData().then(data => {
+    const objArray = [];
+    let feed = [];
+    data.forEach(element => {
+      if (!commentCheck.test(element)) {
+        if (element.startsWith('[')) {
+          // eslint-disable-next-line no-useless-escape
+          line = element.replace(/[\[\]']/g, '');
+          feed.push(`${line}`);
+        }
+        if (nameCheck.test(element)) {
+          line = element.replace(/^\s*name\s*=\s*/, '');
+          feed.push(`${line}`);
+          let obj = {
+            name: feed[feed.length - 1],
+            url: feed[feed.length - 2],
+          };
+          objArray.push(obj);
+          feed = [];
+          obj = {};
         }
-      });
-      return objArray;
-    })
-    .catch(err => {
-      throw err;
```

Comment:
Do we still not need the ```.catch()``` block here since we are using ```.then()```?

Reply:
No, if you leave it off, the caller will get the error.  If all you do is catch and re-throw, you've done the same thing as not having a catch at all.

Reply:
Whoops, for some reason I thought we were logging as well.